### DISCUSSION TITLE
TuringCourses/others/economics/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#166] [General]
# TuringCourses/others/economics/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/others/economics/

**Comment by 626lt** [comment]
写给只想要得到跨专业学分并且不想要花费很多精力的同学：推荐选修政治经济学。这门课完全不点名，前七周是由俞彬老师来上的，讲的很有趣，后9周分别由三个老师来讲一些与政治经济学相关的其他经济学的内容，但相对枯燥。在修这门课的时候只有一开始去了几次，后面由于普物实验和计逻课程的压力就没有再去上过了，但是并不影响最后写论文以及完成作业。这门课的考核要求也是相对宽松的，而且给分很好，完全使用Chat-gpt拟合的同学也能够获得4.5+的分数（x），因此想要水课的同学完全可以冲，任务量不大，严格来说平时的40分作业是一个五分钟的问卷，2个800字以内的思考题和一篇1500字左右的读书报告，合理使用人工智能工具可以在一个下午就完成（x）。期末的论文同样在考试周后截止，作为水课再适合不过了（x）。想要水跨专业学分的同学可以放心冲。
